modules/formulize/class/newsliderElement.php

```diff
@@ -0,0 +1,181 @@
+    }
+
+    // Gather any data to pass to template besides the ele_value
+    // Receives the element object
+    // Returns an array of data that will go to the admin UI template
+    // When dealing with new elements, $element might be FALSE
+    function adminSave($element, $ele_value) {
+        $changed = false;
+        if ($_POST['formlink'] != "none") {
+            $ele_value[4] = $_POST['formlink'];
+        }
+        $element->setVar('ele_value', $ele_value);
+        return $changed;
+    }
+
+    // Reads current state of element, updates ele_value to a renderable state
+    function loadValue($value, $ele_value, $element) {
+        $ele_value[3] = $value;
+        //$ele_value[3] = eregi_replace("'", "&#039;", $ele_value[3]);
+        return $ele_value;
+    }
+
+    // Renders the element for display in a form
+    // Caption is pre-prepared and passed in separately from the element object
+    // If element is disabled return a label with some version of the elements value 
+    // $ele_value contains options for this element
+    // $caption is the prepared caption for the element
+    // $markupName name of rendered element in the HTML
+    // $isDisabled flags whether the element should be rendered or not
+    // $element is the element object
+    // $entry_id is the ID of the entry for the element
+    function render($ele_value, $caption, $markupName, $isDisabled, $element, $entry_id) {
+        $ele_value = $element->getVar('ele_value');
+        $slider_html = "<input type=\"range\" ";
+        $slider_html .= "name=\"{$markupName}\"";
+        $slider_html .= "id=\"{$markupName}\"";
+        $slider_html .= "min=\"{$ele_value[0]}\" ";
+        $slider_html .= "max=\"{$ele_value[1]}\" ";
+        $slider_html .= "step=\"{$ele_value[2]}\" ";
+        $slider_html .= "value=\"{$ele_value[3]}\"";
+        $slider_html .= "oninput=\"updateTextInput(value);\"";
+        $slider_html .= "</input>";
+
+        $value_html = "<output id=\"rangeValue\" type=\"text\" size=\"2\"";
+        $value_html.= "for=\"{$markupName}\"";
+        $value_html .= ">$ele_value[3]<output>";
+
+        $form_slider_value = new XoopsFormLabel($caption, $value_html);
+        $form_slider = new XoopsFormLabel($caption, $slider_html);
+
+        $update_script = "<script type=\"text/javascript\">";
+        $update_script .= "function updateTextInput(val) {";
+        $update_script .= "document.getElementById('rangeValue').value=val;}";
+        $update_script .= "</script>";
+
+        if($isDisabled) {
+            $renderedValue = $form_slider_value->render();
+            $form_ele = new XoopsFormLabel($caption, "$renderedValue");
+        } else {
+            $renderedSlider = $form_slider->render();
+            $renderedValue = $form_slider_value->render();
+            $form_ele = new XoopsFormLabel($caption, "<nobr>$renderedSlider $renderedValue</nobr>.$update_script");
```

Comment:
Can you change ".$update_script" to " $updatescript" because the . shows up on the page?